system/lib/standalone/standalone.c

```diff
@@ -153,7 +153,11 @@ int emscripten_resize_heap(size_t size) {
 }
 
 double emscripten_get_now(void) {
-  return (1000ll * clock()) / (double)CLOCKS_PER_SEC;
+  struct timespec ts;
+  if (clock_gettime(CLOCK_MONOTONIC, &ts)) {
```

Comment:
Why is `clock_gettime` preferable to `clock`? (The monotonicity? But `Date.now` isn't monotonic.)

Reply:
We had a (internal) report of `emscripten_get_now` not increasing while the process was asleep in standalone mode.  

`clock()` is specified as `approximation of processor time used by the program`.  But we want `
`emscripten_get_now` to go up even when the process is asleep.

Internally clock() is based on `CLOCK_PROCESS_CPUTIME_ID`  (https://github.com/emscripten-core/emscripten/blob/5a4814e804d26583a329c8a2fd952046ec72e65f/system/lib/libc/musl/src/time/clock.c#L8) where as here we can explicitly use `CLOCK_MONOTONIC`.

Reply:
I see, thanks.